Sleep Tracker/SleepTracker/Helpers/AppFont.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+
+/// The font used in the application.
```

Comment:
It's more about the size. Would it be better to name it like this?
```suggestion
/// The font size used in the application.
```

Reply:
Yes. Fixed.